### DISCUSSION TITLE
Multiple aws account credential

### DISCUSSION THREAD

**Discussion by prashantrbs** [aws/aws-sdk-js#4461] [Q&A]
Hi There,

I have aws sdk with node js. 

Question: how to use multiple aws account credentials in one sdk

Details: i have 4 aws accounts for 4 different environments like QA, Demo. stage and dev. Right now if I want to use AWS service ofany environment I am pasting aws credential of that environment to .env file. This process i want to change by adding all 4 aws accounts to SDK and then call specific environments creds using some sort of command. i am not sure how i can do this programmatically can someone give me a blog link or any programmatic help? 

Thanks,
Prashant

**Comment by ajredniwja** [comment]
Hi @prashantrbs, I am not sure if I understand the question correctly but this [blog post](https://aws.amazon.com/blogs/security/a-new-and-standardized-way-to-manage-credentials-in-the-aws-sdks/) mentions different ways you can manage your credentials. You can pass in different credential config when instantiating your service client. Which will look something like: 
```js
const aws = require('aws-sdk');
const s3 = new aws.S3({prodConfig}); 
const  s3dev = aws.S3({devConfig})
```
Please reach out if that doesnt help. Thanks
